.vib/osclass/cypress/cypress/support/commands.js

```diff
@@ -0,0 +1,24 @@
+const COMMAND_DELAY = 2000;
+
+for (const command of ['click']) {
+  Cypress.Commands.overwrite(command, (originalFn, ...args) => {
+    const origVal = originalFn(...args);
+
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve(origVal);
+      }, COMMAND_DELAY);
+    });
+  });
+}
+
+Cypress.Commands.add(
+  'login',
+  (username = Cypress.env('username'), password = Cypress.env('password')) => {
+    cy.visit('oc-admin');
```

Comment:
Use a consistent style towards URLs:

```suggestion
    cy.visit('/oc-admin');
```